donation-core/src/main/java/com/donation/Assets.java

```diff
@@ -0,0 +1,7 @@
+package com.donation;
+
+public class Assets {
```

Comment:
It's an enum that could be encapsulated in VideoService as it's linked only to this section

Reply:
Done